README.rst

```diff
@@ -174,6 +175,14 @@ More examples
     with open('s3://bucket/key.txt', 'wb', transport_params=transport_params) as fout:
         fout.write(b'here we stand')
 
+    # stream from GCS
+    for line in open('gs://my_bucket/my_file.txt'):
+        print(line)
+
+    # stream content *into* GCS (write mode):
+    with open('gs://my_bucket/my_file.txt') as fout:
+        fout.write(b'hello world')
```

Comment:
Why are you writing bytes here? Isn't `fout` open in text mode? Also, you're not specifying write mode here. This example looks broken.